src/ThemeProvider.js

```diff
@@ -0,0 +1,89 @@
+import React, {PropTypes, Component} from 'react'
+import mitt from 'mitt'
+
+export const CHANNEL = '__glamorous__'
```

Comment:
The `CHANNEL` key is used in `contextTypes`, I'm not positive that React currently supports using Symbols for keys in context. It would be worth verifying before making this change.